keyboards/inett_studio/sq80/keymaps/via/keymap.json

```
@@ -0,0 +1,24 @@
+{
+    "keyboard": "inett_studio/sq80",
+    "keymap": "via",
+    "layout": "LAYOUT_all",
```

I have a mild preference for using the direct layout name here.

```suggestion
    "layout": "LAYOUT_tkl_ansi",
```

---

Sure thing. I have changed it to use the direct layout name instead.